integration-cli/docker_cli_build_test.go

```diff
@@ -3753,6 +3753,22 @@ ENV    abc 'yyy'
 RUN    [ $abc = \'yyy\' ]
 ENV    abc=
 RUN    [ "$abc" = "" ]
+ENV    abc=\$foo
```

Comment:
I think we have the 2nd one covered here:
https://github.com/docker/docker/blob/master/integration-cli/docker_cli_build_test.go#L3706
but I don't see the 1st - will add it.